cocos/base/allocator/CCAllocatorGlobalNewDelete.cpp

```diff
@@ -28,6 +28,8 @@
 #include <new>
 #include <exception>
 
+#include "base/ccMacros.h"
```

Comment:
This sentence would lead to compile error on the win32 platform for libcocos2d project.Because WinSock2.h is included behind the windows.h.The error message as follows:
C:\Program Files (x86)\Microsoft SDKs\Windows\v7.1A\include\ws2def.h(212): error C2011: 'sockaddr' : 'struct' type redefinition
....
Can we put the `include "base/ccMacros.h"` in the beginning of this file?